kalite/distributed/static/js/distributed/exercises/perseus-helpers.js

```diff
@@ -73,11 +87,11 @@ Exercises.PerseusBridge = {
         }
 
         // Load khan-exercises modules, then perseus
-        require([
+        requirejs([
```

Comment:
Seems like require proxies to some weird pseudo-require that KA have bundled somewhere, and that doesn't quite work.